scripts/gitbook.js

```diff
@@ -9,6 +9,63 @@ const path = require('path')
  */
```

Comment:
We should run this as part of the github push/lint workflow. It can be a separate task, if you want you can create it and assign it to me :)